core/src/test/scala/org/apache/spark/scheduler/TaskSchedulerImplSuite.scala

```diff
@@ -73,17 +73,15 @@ class TaskSchedulerImplSuite extends SparkFunSuite with LocalSparkContext with B
     }
   }
 
-  def setupScheduler(confs: (String, String)*): TaskSchedulerImpl = {
+  private def setupScheduler(confs: (String, String)*): TaskSchedulerImpl = {
```

Comment:
one quick comment here if Imran didn't already merge: can you un-do these changes?  It's not useful / necessary to make test classes private (they're already hidden by the build), and this change will make git blames more confusing in the future.